presto-main/src/test/java/com/facebook/presto/security/TestFileBasedSystemAccessControl.java

```diff
@@ -39,20 +40,26 @@
     private static final Identity alice = new Identity("alice", Optional.of(PRINCIPAL));
     private static final Identity bob = new Identity("bob", Optional.of(PRINCIPAL));
     private static final Identity admin = new Identity("admin", Optional.of(PRINCIPAL));
-    private static final Identity nonAsciiUser = new Identity("ÎÎÎ", Optional.of(PRINCIPAL));
-    private static final Set<String> allCatalogs = ImmutableSet.of("secret", "open-to-all", "all-allowed", "alice-catalog", "allowed-absent", "ÅÅÅ");
+    private static final Identity nonAsciiUser = new Identity("\u0194\u0194\u0194", Optional.of(PRINCIPAL));
+    private static final Set<String> allCatalogs = ImmutableSet.of("secret", "open-to-all", "all-allowed", "alice-catalog", "allowed-absent", "\u0200\u0200\u0200");
     private static final QualifiedObjectName aliceTable = new QualifiedObjectName("alice-catalog", "schema", "table");
     private static final QualifiedObjectName aliceView = new QualifiedObjectName("alice-catalog", "schema", "view");
     private static final CatalogSchemaName aliceSchema = new CatalogSchemaName("alice-catalog", "schema");
+    private TransactionManager transactionManager;
+
+    @BeforeClass
```

Comment:
If you make this BeforeTest, you can also move this there:
AccessControlManager accessControlManager = new AccessControlManager(transactionManager);
accessControlManager.setSystemAccessControl(FileBasedSystemAccessControl.NAME, ImmutableMap.of(
"security.config-file", this.getClass().getClassLoader().getResource("catalog.json").getPath()));